hw/top_earlgrey/data/ip/chip_aes_testplan.hjson

```diff
@@ -0,0 +1,259 @@
+            -Using dif_aes_read_iv read the IV. Note that AES has automatically updated the content
+             of this register. This IV will be needed later to continue this encryption.
+            -De-initialize automatic operation. This step is necessary to clear all registers.
+            -Perform an encryption using a different data and key. Message size, key size and mode
+             can be chosen arbitrarily.
+            -Continue the original encryption:
+                -Configure the operation using the previously read IV
+                -Process the remaining plaintext blocks using dif_aes_process_data
+            -Decrypt the result (using the original key and IV)
+            -Check that the decryption result matches the original plaintext.
+            '''
+      features: [
+        "AES.MODE.CBC",
+        "AES.MODE.CFB-128",
+        "AES.MODE.OFB",
+        "AES.MODE.CTR",
+      ]
+      stage: V2
+      si_stage: SV3
+      lc_states: ["PROD"]
+      tests: []
+    }
+    {
+      name: chip_sw_aes_entropy
+      desc: '''Verify the AES entropy input used by the internal PRNGs.
+
+            - Write the initial key share, IV and data in CSRs (known combinations).
+            - Configure the entropy_src to generate entropy in LFSR mode.
+            - Write the PRNG_RESEED bit to reseed the internal state of the PRNG.
+            - Poll the status idle bit to ensure reseed operation is complete.
+            - Trigger the AES operation to run and wait for it to complete.
+            - Check the digest against the expected value.
+            - Write the KEY_IV_DATA_IN_CLEAR and DATA_OUT_CLEAR trigger bits to 1 and wait for it to
+              complete by polling the status idle bit.
+            - Read back the data out CSRs - they should all read garbage values.
+            - Assertion check verifies that the IV are also garbage, i.e. different from the
+              originally written values.
+            '''
+      features: [
+        "AES.MODE.ECB",
+        "AES.CLEAR.DATA_OUT",
+        "AES.CLEAR.KEY_IV_DATA_IN",
+      ]
+      stage: V2
+      si_stage: SV2
+      lc_states: ["PROD"]
+      tests: ["chip_sw_aes_entropy"]
+    }
+    {
+      name: chip_sw_aes_prng_reseed
+      desc: '''Verify that prng reseeding is triggered
+
+            This test should start an encryption of a long message, disable the EDN complex, and
+            verify that the the encryption stalls when reseed is required.
+
+            Procedure:
+            -Configure encryption using the desired mode, key length, IV, and set the engine to
+            automatic operation.
+                -Set the mask_reseeding to once per 64 blocks (kDifAesReseedPer64Block)
+            -Start encryption of a long message (longer than 64 blocks)
+            -Disable the EDN complex using dif_edn_stop().
+            -Verify that the encryption stalls after 64 blocks are encrypted.
```

Comment:
What should be added here is to enable the full entropy complex again (note that there is specified order regarding how to enable the EDNs, CSRNG etc. you cannot just disable EDN and then re-enable it again later) to verify that AES unstalls and finishes the last block.

Reply:
OK. I've added re-enabling of the entropy complex to the descriptions